mne/io/fieldtrip/tests/helpers.py

```diff
@@ -142,26 +144,33 @@ def get_raw_data(system, drop_sti_cnt=True, drop_extra_chs=False):
 def get_epochs(system):
     """Find, load and process the epoched data."""
     cfg_local = get_cfg_local(system)
-    raw_data = get_raw_data(system, drop_sti_cnt=False)
+    raw_data = get_raw_data(system)
 
     if cfg_local['eventtype'] in raw_data.ch_names:
         stim_channel = cfg_local['eventtype']
     else:
         stim_channel = 'STI 014'
 
-    events = mne.find_events(raw_data, stim_channel=stim_channel,
-                             shortest_event=1)
+    if system == 'CNT':
+        events, event_id = mne.events_from_annotations(raw_data)
+    else:
+        events = mne.find_events(raw_data, stim_channel=stim_channel,
+                                 shortest_event=1)
 
     if system == 'CNT':
-        raw_data.drop_channels(['STI 014'])
         events[:, 0] = events[:, 0] + 1
 
-    if isinstance(cfg_local['eventvalue'], np.ndarray):
-        event_id = list(cfg_local['eventvalue'].astype('int'))
-    else:
-        event_id = [int(cfg_local['eventvalue'])]
+    if 'event_id' not in locals():
```

Comment:
calling locals() here seems dangerous makes the code harder to read. Can you see how to avoid it? thx

Reply:
forget it it's in tests. Not strongly necessary then but just nice to do.